docs/release/3.5/release-3-5-0.md

```diff
@@ -0,0 +1,145 @@
+OSG Software Release 3.5.0
+===========================
+
+**Release Date:** 2019-08-30    
+**Supported OS Versions:** EL7
+
+Summary of changes
+------------------
+
+This initial release of the OSG 3.5 release series is based on the packages available in
+[OSG 3.4.33](/release/3.4/release-3-4-33#) with some [additions](#package-updates) and [subtractions](#package-removals).
+Other notable changes in this release series include dropping support for Enterprise Linux 6 and
+[CREAM CEs](https://opensciencegrid.org/technology/policy/cream-support/).
+
+To update to the OSG 3.5 release series, please consult the page on
+[updating between release series](/release/release_series#updating-to-osg-35).
+
+!!! question "Where are GlideinWMS and HTCondor-CE?"
+    HTCondor-CE (including `osg-ce` metapackages) and GlideinWMS are both absent in OSG 3.5.0:
+    we expect major version upgrades that may require manual intervention for both of these packages so we are holding
+    their initial releases in this series until they are ready.
+
+!!! warning "OSG 3.4 end-of-life"
+    As a result of this initial OSG 3.5 release, the end-of-life dates have been set for OSG 3.4 per our
+    [policy](https://opensciencegrid.org/technology/policy/release-series/):
+    regular support will end in **February 2020** and critical bug/security support will end in **November 2020**.
+
+### Package updates ###
+
+In addition to the packages that were carried over from [OSG 3.4.33](/release/3.4/release-3-4-33#),
+this release contains the following package updates:
+
+-   HTCondor 8.8.4: See the [manual upgrade instructions](/release/release_series#updating-to-htcondor-8.8.x) before
```

Comment:
@timtheisen can you add a few words here mentioning that this is a major version upgrade from previously distributed versions in the OSG + a few highlights of 8.8?

Reply:
- Automatically add AWS resources to your pool using HTCondor Annex
- The Python bindings now include submit functionality
- Added the ability to run a job immediately by replacing a running job
- A new mini(ht)condor package makes single node installations easy
- HTCondor now tracks and reports GPU utilization
- Several performance enhancements in the collector
- The grid universe can create and manage VM instances in Microsoft Azure
- The MUNGE security method is now supported on all Linux platforms